README.md

```diff
@@ -141,34 +144,27 @@ Wait until all tasks are running before trying to access the Kubernetes API.
 You can watch the progress what was deployed so far with:
 
 ```bash
-$ watch ./dcos kubernetes plan show deploy
+$ make watch
 ```
 
 Below is an example of how it looks like when the install ran successfully:
 
 ```
+Using Kubernetes cluster: dev/kubernetes01
 deploy (serial strategy) (COMPLETE)
    etcd (serial strategy) (COMPLETE)
       etcd-0:[peer] (COMPLETE)
-   apiserver (dependency strategy) (COMPLETE)
-      kube-apiserver-0:[instance] (COMPLETE)
+   control-plane (dependency strategy) (COMPLETE)
+      kube-control-plane-0:[instance] (COMPLETE)
    mandatory-addons (serial strategy) (COMPLETE)
-      mandatory-addons-0:[additional-cluster-role-bindings] (COMPLETE)
-      mandatory-addons-0:[kubelet-tls-bootstrapping] (COMPLETE)
-      mandatory-addons-0:[kube-dns] (COMPLETE)
-      mandatory-addons-0:[metrics-server] (COMPLETE)
-      mandatory-addons-0:[dashboard] (COMPLETE)
-      mandatory-addons-0:[ark] (COMPLETE)
-   kubernetes-api-proxy (dependency strategy) (COMPLETE)
-      kubernetes-api-proxy-0:[install] (COMPLETE)
-   controller-manager (dependency strategy) (COMPLETE)
-      kube-controller-manager-0:[instance] (COMPLETE)
-   scheduler (dependency strategy) (COMPLETE)
-      kube-scheduler-0:[instance] (COMPLETE)
+      mandatory-addons-0:[instance] (COMPLETE)
    node (dependency strategy) (COMPLETE)
-      kube-node-0:[kube-proxy, coredns, kubelet] (COMPLETE)
+      kube-node-0:[kubelet] (COMPLETE)
    public-node (dependency strategy) (COMPLETE)
```

Comment:
same as above `serial strategy`

Reply:
@hectorj2f this is actually accurate, as reported by `dcos kubernetes cluster debug` 🤔

Reply:
okay, I know why it differs, the cluster has been updated. Let's keep it as it is.

Reply:
To give some context for the rest of the team, when you perform an update. It uses a different plan (the update plan) where the strategy is `serial` for some pods.